.hopper/config.yml

```diff
@@ -1,6 +1,22 @@
 version: '1'
 
 services:
+  web:
+    containerDefinitions:
+      worker:
```

Comment:
is `worker` the right name to use here? maybe template this for the internal-web worker too?